### ISSUE TITLE
Aliases to run the script from a shell

### ISSUE THREAD

**Issue by matiscke** [matiscke/scan_astro-ph#12]
Currently there are three aliases that can be used to call the program:
- scan_astro-ph
- scan_astroph
- astroph-scan

This can be a bit cumbersome for Tab-completion in the shell, as one gets stuck where multiple option to continue exist (at several points in case of the first two aliases).

Maybe we should agree on a single command to invoke scan_astro-ph and drop the aliases.

**Comment by Simske** [comment]
Yes, I totally agree. I wanted to already choose them in #5, but forgot about it.

I feel like underscores are more cumbersome to type an underscore, and beginning with `astroph` help tab-completion. Therefore I prefer `astroph-scan`.

As could be easily extended to other topics on arXiv.org, maybe we could also think about renaming it to `arxiv-scan` or something similar.

**Comment by matiscke** [comment]
Agreed on the `astroph-scan`. 

I like the idea of renaming to `arxiv-scan`, _if_ we can make it straightforward to the user to pick their arxiv section. This will probably be done only once. Some users might want to cover several (sub)sections. 

In any case, `arxiv-scan` would be more general and also good for tab completion.

**Comment by Simske** [comment]
Sounds good. What do you think about changing it to `astroph-scan` for now, and add `arxiv-scan` when topic selection is implemented?

**Comment by Simske** [comment]
Also: should we rename the Python package? This doesn't make any difference for the user, but I might be nice to have this unified.

**Comment by matiscke** [comment]
Yes, I think it would be good if it's all the same name.
